tests/rc/models/transformer_qa_test.py

```diff
@@ -44,6 +44,10 @@ def test_forward_pass_runs_correctly(self):
         # script.
         assert metrics["per_instance_f1"] > 0
 
+        output_span_start_probs = output_dict["span_start_probs"][0].data.numpy()
+        output_span_end_probs = output_dict["span_end_probs"][0].data.numpy()
+        assert_almost_equal(numpy.sum(output_span_start_probs, -1), 1, decimal=6)
+        assert_almost_equal(numpy.sum(output_span_end_probs, -1), 1, decimal=6)
```

Comment:
We probably can't do this style of test for `best_span_probs`, because we don't want to enumerate all possible spans, but we can test some basics like "They should all be greater than 0." and "Their sum must be <= 1.".

Though I'm looking at this through a cardboard tube view. If I'm missing something that makes this hard, don't bend over backwards to make it happen.

Reply:
Done. It didn't seem too hard to add those checks, let me know if you think of anything else.